File: ProApi.App/src/app/_models/Event.ts
import { Lot } from './Lot';
import { SocialNetwork } from './SocialNetwork';
import { Speaker } from './Speaker';

export class Event {

  constructor() { }

  id: number;
  place: string;
  dateEvent: Date;
  theme: string;
  qtdPerson: number;
  imageUrl: string;
  phone: string;
  email: string;
  lots: Lot[];
  socialNetworks: SocialNetwork[];
  speakerEvents: Speaker[];
}

File: ProApi.Domain/Event.cs
using System;
using System.Collections.Generic;

namespace ProApi.Domain
{
    public class Event
    {
        public int Id { get; set; }    
        public string Place { get; set; }   
        public DateTime DateEvent { get; set; }
        public string Theme { get; set; }
        public int QtdPerson { get; set; }
        public string ImageUrl { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public List<Lot> Lots { get; set; }
        public List<SocialNetwork> SocialNetworks { get; set; }
        public List<SpeakerEvent> SpeakerEvents { get; set; }
    }
}